packages/react/src/components/Select/Select.tsx

```diff
@@ -255,6 +258,13 @@ const Select = React.forwardRef(function Select(
       size: 'mini',
     });
   }
+  useEffect(() => {
+    const selectElement = document.getElementById(
+      id
+    ) as HTMLSelectElement | null;
+    setTitle(selectElement?.options[selectElement?.selectedIndex]?.text || '');
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
This needs to run on every render of the component? If so, I don't think it needs to be in a `useEffect`.  If the `select` has a `ref` on it, can we use the ref to get the selected index/element instead of using document.getElementById?

We have a `mergeRefs` helper function that can be used to combine the forwarded `ref` and a local ref inside the component for this use if it's needed.